specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/instanceFailoverGroups.json

```diff
@@ -0,0 +1,644 @@
+          "readOnly": true
+        },
+        "partnerRegions": {
+          "description": "Partner region information for the failover group.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PartnerRegionInfo"
+          },
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "managedInstancePairs": {
+          "description": "List of managed instance pairs in the failover group.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagedInstancePairInfo"
+          }
+        }
+      }
+    },
+    "InstanceFailoverGroupReadWriteEndpoint": {
+      "description": "Read-write endpoint of the failover group instance.",
+      "required": [
+        "failoverPolicy"
+      ],
+      "type": "object",
+      "properties": {
+        "failoverPolicy": {
+          "description": "Failover policy of the read-write endpoint for the failover group. If failoverPolicy is Automatic then failoverWithDataLossGracePeriodMinutes is required.",
+          "enum": [
+            "Manual",
+            "Automatic"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ReadWriteEndpointFailoverPolicy",
+            "modelAsString": true
+          }
+        },
+        "failoverWithDataLossGracePeriodMinutes": {
+          "format": "int32",
+          "description": "Grace period before failover with data loss is attempted for the read-write endpoint. If failoverPolicy is Automatic then failoverWithDataLossGracePeriodMinutes is required.",
+          "type": "integer"
+        }
+      }
+    },
+    "InstanceFailoverGroupReadOnlyEndpoint": {
+      "description": "Read-only endpoint of the failover group instance.",
+      "type": "object",
+      "properties": {
+        "failoverPolicy": {
+          "description": "Failover policy of the read-only endpoint for the failover group.",
+          "enum": [
+            "Disabled",
+            "Enabled"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ReadOnlyEndpointFailoverPolicy",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "PartnerRegionInfo": {
+      "description": "Partner region information for the failover group.",
+      "type": "object",
+      "properties": {
+        "location": {
+          "description": "Geo location of the partner managed instances.",
+          "type": "string"
+        },
+        "replicationRole": {
+          "description": "Replication role of the partner managed instances.",
+          "enum": [
+            "Primary",
+            "Secondary"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "InstanceFailoverGroupReplicationRole",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "ManagedInstancePairInfo": {
```

Comment:
Can this be changed to just ManagedInstancePair ? "Info" seems redundant. All objects are information about something.